tools/reviewer/job/sync/GitHubWriter.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.reviewer.job.sync;
+
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.CreatePullRequestCommentRequest;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.CreatePullRequestCommentRequestData;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.CreatePullRequestRequest;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.CreatePullRequestRequestData;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.CreatePullRequestResponse;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.CreateReviewCommentRequest;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.CreateReviewCommentRequestData;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.PullRequestRequest;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.File;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.PullRequest;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.PullRequestFilesRequest;
+import com.google.startupos.tools.reviewer.job.sync.GitHubProtos.PullRequestsRequest;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.Comment;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.Diff;
+import com.google.startupos.tools.reviewer.localserver.service.Protos.Thread;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+
+public class GitHubWriter {
+  private final GitHubClient gitHubClient;
+
+  GitHubWriter(GitHubClient gitHubClient) {
+    this.gitHubClient = gitHubClient;
+  }
+
+  public void writeDiff(Diff diff, String repo) throws IOException {
+    int diffNumber = getPRNumber(diff, repo);
+
+    for (Thread diffThread : diff.getDiffThreadList()) {
+      for (Comment comment : diffThread.getCommentList()) {
+        // TODO: Think over what should be in the comment by Reviewer Bot
```

Comment:
Can we add a link to the Reviewer Diff, as a text link (e.g with the text "see in Reviewer")?

Reply:
Yes, we can add the link to the Reviewer Diff (e.g. https://startupos-5f279.firebaseapp.com/diff/65/startup-os/tools/reviewer/service/CodeReviewService.java). But it won't be the link to the concrete line.